MaxText/train.py

```diff
@@ -94,15 +94,15 @@ def write_metrics(writer, metrics, step, config):
       for metric_name in metrics.get("scalars",[]):
         writer.add_scalars(metric_name, metrics["scalars"][metric_name], step)
 
-    full_log = step % config.log_period == 0
+    full_log = step % 5 == 0
```

Comment:
I don't love adding more constants scatter around the codebase and not sure I fully understand this particular one. Does the `log_period` only influence this or are there other consumers? If there aren't any other consumers, maybe we just set the default `log_period` to say 10?